ui/client/components/SWRHooks.js

```diff
@@ -205,7 +205,11 @@ export function useDocuments(scrollId) {
   }
 
   const { data, error, mutate } = useSWR(
-    url, fetcher
+    url, fetcher, {
+      revalidateIfStale: false,
```

Comment:
Clicking out and back to the browser window was forcing a refetch. We don't want this, as we may have performed a semantic search and may be displaying search results (we don't want those results to be replaced by a full documents refetch).